lib/plugins/usermanager/lang/it/lang.php

```diff
@@ -66,8 +66,6 @@
 $lang['add_fail']              = 'Aggiunta utente fallita';
 $lang['notify_ok']             = 'Email di notifica inviata';
 $lang['notify_fail']           = 'L\'email di notifica non può essere inviata';
-$lang['import_userlistcsv']    = 'File lista utente (CSV):';
```

Comment:
How can this happen (removing something from a translation)?
As far as I can tell, it's still a valid entry.